### PR TITLE
UA: Surface active machine count

### PR DESCRIPTION
**Pull request by jpmartinspt**
## Done

Surface machine activity numbers to front-end

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes #https://github.com/canonical-web-and-design/commercial-squad/issues/448
Fixes #https://github.com/canonical-web-and-design/commercial-squad/issues/454

### CODE REVIEW

**Comment by albertkol** [comment]
Is there any way I can activate a machine? I always have 0 active machines. @jpmartinspt

**Review by albertkol — APPROVED** [review]
:+1:

File: static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/DetailsContent/DetailsContent.tsx [hunk 1]
```diff
@@ -143,6 +146,31 @@ const DetailsContent = ({ selectedId }: Props) => {
...
+              []
+            : [
+                {
+                  title: "Active machines",
+                  value: (
+                    <React.Fragment>
+                      {subscription.number_of_active_machines}
+                      <Tooltip
+                        tooltipClassName="p-subscriptions-tooltip"
+                        message="The number of machines that contacted Ubuntu Advantage in the last 24 hour period"
```
Comment (wgx) [inline static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/DetailsContent/DetailsContent.tsx:160]: (Beta)

Comment (jpmartinspt) [reply static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/DetailsContent/DetailsContent.tsx]: @wgx made the change, you can check on the demo and if it's good we can merge.

**Comment by wgx** [comment]
@jpmartinspt The demo isn't available so I can't review.

**Comment by wgx** [comment]
@jpmartinspt +1 lgtm

**Pull request merged**